plugins/woocommerce/client/legacy/css/forms.scss

```diff
@@ -0,0 +1,154 @@
+.woocommerce {
+	/**
+	 * Generic forms styles used in places such as my account and the shortcode based checkout.
+	 */
+	form .form-row {
+		padding: 3px;
+		margin: 0 0 6px;
+
+		[placeholder]:focus::-webkit-input-placeholder {
+			transition: opacity 0.5s 0.5s ease;
+			opacity: 0;
+		}
+
+		label {
+			line-height: 2;
+		}
+
+		label.hidden {
+			visibility: hidden;
+		}
+
+		label.inline {
+			display: inline;
+		}
+
+		.woocommerce-input-wrapper {
+			.description {
+				background: #1e85be;
+				color: #fff;
+				border-radius: 3px;
+				padding: 1em;
+				margin: 0.5em 0 0;
+				clear: both;
+				display: none;
+				position: relative;
```

Comment:
are these styles all migrated from other places? odd to see position set when display is none.